spec/helpers/spec-helper.coffee

```diff
@@ -33,40 +45,48 @@ beforeEach ->
   $.fx.off = true
 
   # Basic page fixture
-  $('#jasmine_content').html("<div id='wrapper'></div><div id='overlays'></div><div id='side-nav'></div><div id='main-view'></div></div>")
-
-  # Setup a new base.
-  window.base = {}
-  window.base.data = new Brainstem.StorageManager()
-  window.base.data.addCollection 'time_entries', App.Collections.TimeEntries
-  window.base.data.addCollection 'posts', App.Collections.Posts
-  window.base.data.addCollection 'tasks', App.Collections.Tasks
-  window.base.data.addCollection 'projects', App.Collections.Projects
-  window.base.data.addCollection 'users', App.Collections.Users
-
+  $(document.body).html('''
+    <div id="jasmine_content">
+      <div id="wrapper"></div>
+      <div id="overlays"></div>
+      <div id="side-nav"></div>
```

Comment:
I really doubt it since brainstem-js should be all about data and not views. 

Quick delete and + specs? Perhaps also check out if the specs pass without these `div`s in current master (but commit here so less conflicts, less headaches)?